docs/introduction.md

```diff
@@ -17,19 +17,31 @@ doc-topic: [introduction]
 Kubewarden is a Kubernetes Policy Engine.
 It aims to be the Universal Policy Engine for Kubernetes.
 
-- You can reuse policies from other policy engines without having to rewrite
-them.
-- You can write your own policies in any programming language that generates
-WebAssembly binaries, reusing your language tooling and libraries.
-- You can run policies both outside the cluster and as part of your CI/CD
-processes.
-
-Kubewarden also provides an audit scanner to actively and continuously check
-policy enforcement over time.
-
 Kubewarden is a [CNCF](https://cncf.io) Sandbox project,
 originally created by [SUSE Rancher](https://www.rancher.com/).
 
+## How does Kubewarden help?
+
+Kubewarden offers flexibility for policy admission and enforcement in a Kubernetes environment.
+
+### Benefits and value
+
+- Use any programming language that generates [WebAssembly](https://webassembly.org) binaries to write your policies.
+- Utilizing WebAssembly allows policy compatibility across processors and operating systems.
```

Comment:
```suggestion
- WebAssembly enables policy compatibility across processors and operating systems.
```

Reply:
Here I suggested rephrasing to match the other items in the list that do not use a gerund.